app/script/config.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Wire
+ * Copyright (C) 2016 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+if (window.z === null) {
+  window.z = {};
+}
+
+z.config = {
+  BROWSER_NOTIFICATION: {
+    TIMEOUT: 5000,
+    TITLE_LENGTH: 38,
+    BODY_LENGTH: 80,
+  },
+
+  LOGGER: {
+    OPTIONS: {
+      name_length: 65,
+      domains: {
+        'app.wire.com'() {
+          return 0;
+        },
+        'localhost'() {
+          return 300;
+        },
+        'wire.ms'() {
+          return 300;
+        },
+        'wire-webapp-staging.wire.com'() {
+          return 300;
+        },
+        'zinfra.io'() {
+          return 300;
+        },
+      },
+    },
+  },
+
+  // number of message that will be pulled
+  MESSAGES_FETCH_LIMIT: 30,
+
+  // number of users displayed in people you may know
+  SUGGESTIONS_FETCH_LIMIT: 30,
+
+  // Accent color IDs
+  ACCENT_ID: {
+    BLUE: 1,
+    GREEN: 2,
+    YELLOW: 3,
+    RED: 4,
+    ORANGE: 5,
+    PINK: 6,
+    PURPLE: 7,
+  },
+
+  // Conversation size
+  MAXIMUM_CONVERSATION_SIZE: 128,
+
+  // self profile image size
+  MINIMUM_PROFILE_IMAGE_SIZE: {
+    WIDTH: 320,
+    HEIGHT: 320,
+  },
+
+  // 15 megabyte image upload limit
+  MAXIMUM_IMAGE_FILE_SIZE: 15 * 1024 * 1024,
+
+  // 25 megabyte upload limit ( minus iv and padding )
+  MAXIMUM_ASSET_FILE_SIZE: 25 * 1024 * 1024 - 16 - 16,
+
+  // Maximum of parallel uploads
+  MAXIMUM_ASSET_UPLOADS: 10,
+
+  // Maximum characters per message
```

Comment:
No, because it explains that the length is measured in characters, not in bytes or anything else.